<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cMovieEdit</class>
 <widget class="QDialog" name="cMovieEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="m_lpName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Year:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="m_lpFirstAired">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="m_lpInit">
           <property name="text">
            <string>Initialized</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="m_lpProgress">
           <property name="text">
            <string>in Progress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="m_lpDone">
           <property name="text">
            <string>done</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="m_lpDetailsBanner">
       <property name="minimumSize">
        <size>
         <width>310</width>
         <height>200</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>785</width>
        <height>310</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTextEdit" name="m_lpDetailsOverview">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="cVerticalLabel" name="label_2">
               <property name="text">
                <string>Actors</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeWidget" name="m_lpDetailsActors">
               <property name="columnCount">
                <number>2</number>
               </property>
               <attribute name="headerVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string notr="true">2</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="cVerticalLabel" name="label_3">
               <property name="text">
                <string>Genre</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeWidget" name="m_lpDetailsGenre">
               <property name="columnCount">
                <number>2</number>
               </property>
               <attribute name="headerVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string notr="true">2</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cVerticalLabel</class>
   <extends>QLabel</extends>
   <header>cverticallabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cMovieEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cMovieEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
